compiler/luci/pass/src/QuantizeDequantizeWeightsPass.cpp

```diff
@@ -106,6 +113,27 @@ void asym_wquant_with_minmax(CircleConst *node, float min, float max, float *sca
   }
 }
 
+void asym_wdequant_with_minmax(CircleConst *node, float scaling_factor, float nudged_min)
+{
+
+  uint32_t size = node->size<loco::DataType::U8>();
+  std::vector<float> dequantized_values(size);
+  for (int i = 0; i < static_cast<int32_t>(size); ++i)
```

Comment:
How about this change? `size` shouldn't be minus value I think.
```suggestion
  for (uint32_t i = 0; i < size; ++i)
```